books/free-programming-books-ja.md

```diff
@@ -338,6 +339,11 @@
 * [図解 Git](https://marklodato.github.io/visual-git-guide/index-ja.html) - Mark Lodato, Kazu Yamamoto(翻訳)
 
 
+### GNU-Make
+
+* [GNU Make 第3版](https://www.oreilly.co.jp/library/4873112699/) - Robert Mecklenburg(翻訳)
```

Comment:
It seems that there are to pay for it

![image](https://user-images.githubusercontent.com/3125580/135697161-2fe1d684-8e4d-436b-b462-9bb7286f2c32.png)